contents/en/rules_en.md

```diff
@@ -1,52 +1,52 @@
-|0. Administration applies these rules on its discretion.|
+|0. Administration applies these rules at their discretion.|
 | :------------ |
-|That means that Administration is free not to punish for violations if it's necessary, or if there's no grievances to the offender.|
+|That means that Administration is free not to punish for violations if it's necessary, or if there are no grievances to the offender.|
 
 |1.  Don't be an asshole.|
 | :------------ |
-|This is just a game, and we here for enjoyment. Long-timed conflicts, intentional harm to community, toxic behavior are not allowed, and will be punished. This rule doesn't regulate minor conflicts.|
+|This is just a game, and we here for enjoyment. Long-term conflicts, intentional harm to the community, and toxic behavior are not allowed and will be punished. This rule does not apply to minor conflicts.|
 
 |2.  Please, do your job.|
 | :------------ |
-|Engine must be started in correct times, people shouldn't die in medbay's reception while doctors are having tea, and so on. Remember, that the safety and status of the whole ship are depend of crucial roles' work. Heads and AI must not leave after round's start. But if you have your serious reasons which does not allow you to work, it's ok.|
+|Engine must be started at the correct time, people shouldn't die in medbay's reception while doctors are having tea, and so on. Remember, that the safety and status of the whole ship are dependant on the work of crucial roles. Heads and AI must not leave after the round's start. But if you have a serious reason which prevents you from doing your job, it's ok to leave.|
 
-|3.  Multi accounting.|
+|3.  Multi-accounting.|
 | :------------ |
-|This disrupts Administration's work, and punishes hard. If you want to change your account, you should discuss it with the Administration. Admins can use multi accounts for their purposes.|
+|This disrupts Administration's work and is punished harshly. If you want to change your account, you should discuss it with the Administration. Admins can use multi accounts for their purposes.|
 
 |4.  Intentional murder/isolation/infliction of grievous bodily harm without any reason.|
 | :------------ |
-|You should have strong cause for killing. It can be: selfdefence, long-time roleplayed conflict, the fulfillment of the order(The one who gave an order to you will be responsible) and the enforcement of the sentence.|
-|The violation always will be considerated in context of events, which were earlier. Usually, the one who generate the situation is the one who takes responsibility.|
-|Full isolation of the player = kill.|
-|Infliction of grievous bodily harm, which causes long healing in medbay (more than 30 minutes) punishes in half.|
+|You should have a strong case for killing. It can be self-defense, a conflict roleplayed over a long period of time, the fulfillment of a order (The one who gave an order to you will be responsible) or the enforcement of a sentence.|
```

Comment:
I feel like it was supposed to be "cause" not "case". As in, "a strong reason for killing", not "a strong situation for killing".

Reply:
Also, "the flfillment of an order", instead of "a order".